src/components/AppContainer/AppContainer.tsx

```diff
@@ -15,7 +15,7 @@ export default class AppContainer extends React.Component<NestableBaseComponentP
     const { children, className, theme = defaultTheme } = this.props;
     const classNames = getClassNames({ theme });
     return (
-      <Fabric>
+      <Fabric theme={theme}>
```

Comment:
might need to double check this when you merge from master as Donnie's PR has been merged

Reply:
Yeah, I see it's conflicting.  I'll resolve it.